components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1931,6 +1917,94 @@ void RewardsServiceImpl::SaveInlineMediaInfo(
                     std::move(callback)));
 }
 
+void RewardsServiceImpl::UpdateMediaDuration(
+    const uint64_t window_id,
+    const std::string& publisher_key,
+    const uint64_t duration) {
+  if (!Connected()) {
+    return;
```

Comment:
this should callback as extension is expecting response

Reply:
There's no callback for `UpdateMediaDuration`.

Reply:
ah, right